### ISSUE TITLE
Identify on every page

### ISSUE THREAD

**Issue by sperand-io** [segmentio/analytics-magento#64]
Currently, we're only calling `identify` on the login or signup events (from what I can tell? - please correct me if I'm wrong :).

A segment implementation best practice is to call identify on every page load if there is a signed-in user present. That way, if a customer clicks an email link to their shopping cart that signs them in automatically, or if a customer returns to the store and their session is still active, we'll have all their traits cached in analytics.js to pass with subsequent track and page calls.

TLDR: We should call `identify` conditionally right above `page` in the snippet if there is a logged in user present. 

@astorm cc @ianstormtaylor

**Comment by reinpk** [comment]
+1 identifying everywhere is the best practice we promote in general, and solves some key use cases for email marketing, which is a huge channel for all the ecommerce companies i've spoken to.

**Comment by sperand-io** [comment]
Fixed :)
